src/MagnumPlugins/AssimpImporter/AssimpImporter.conf

```diff
@@ -0,0 +1 @@
+depends=AnyImageImporter
```

Comment:
More of a reminder for myself (I can do that post-merge):

There should be a bunch of `provides=` lines here, based on what all formats this supports (at least it should alias *all* the existing OBJ, COLLADA, Stanford, .. importers), mentioning this also in the docs and then finally updating `AnyImporter` to detect and handle the new formats.